arcgis-ios-sdk-samples/Scenes/Display 3D labels in scene/Display3DLabelsSceneViewController.swift

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class Display3DLabelsSceneViewController: UIViewController {
+    @IBOutlet var sceneView: AGSSceneView! {
+        didSet {
+            // Set the scene to the scene view.
+            sceneView.scene = makeScene()
+        }
+    }
+    
+    func makeScene() -> AGSScene {
+        let scene = AGSScene(
+            item: AGSPortalItem(
+                portal: AGSPortal.arcGISOnline(withLoginRequired: false),
+                itemID: "850dfee7d30f4d9da0ebca34a533c169")
+        )
+        // Load the scene.
+        scene.load { [weak self] error in
+            // Get the feature layer.
+            if let layers = scene.operationalLayers as? [AGSGroupLayer],
+               let groupLayer = layers.first(where: { $0.name == "Gas" }),
+               let gasFeatureLayer = groupLayer.layers.firstObject as? AGSFeatureLayer {
+                do {
+                    guard let labelDefinition = try self?.makeLabelDefinition() else { return }
+                    // Enable labels on the feature layer.
+                    gasFeatureLayer.labelsEnabled = true
+                    // Add the label definition to the layer.
+                    gasFeatureLayer.labelDefinitions.add(labelDefinition)
+                } catch {
+                    // If failure to make a label definition, present an error.
+                    print(error)
+                }
+            }
+        }
+        return scene
+    }
+    
+    func makeLabelDefinition() throws -> AGSLabelDefinition? {
+        // Make and stylize the text symbol.
+        let textSymbol = AGSTextSymbol()
```

Comment:
The minimal set of properties to achieve the exactly same visual appearance: 

```swift
        let textSymbol = AGSTextSymbol()
        textSymbol.color = .orange
        textSymbol.haloColor = .white
        textSymbol.haloWidth = 2
        textSymbol.size = 14
```

---

Paste my question from the design review:

I'm a little bit curious which properties we should showcase in this sample. It is totally fine to use what we have here, but...

- maybe we should only demo the properties that we set to non-default values
    This is arguable. We may want to demonstrate all the available properties for a text symbol (which we have more properties here), or we might not want to let the creation of text symbol dilute the purpose to demo 3D symbols in a scene (which we use less).
- what is the difference between `outlineColor` and `haloColor`? 
    It doesn't look different in this sample.
- what about using a factory method/closure block to generate the text symbol?